climada/hazard/test/test_tc_tracks.py

```diff
@@ -622,6 +622,13 @@ def test_interp_track_pass(self):
             with self.assertRaises(ValueError, msg=msg) as _cm:
                 tc_track.equal_timestep(time_step_h=time_step_h)
 
+        # test for tracks with non-normalized longitude
+        tc_track = tc.TCTracks.from_ibtracs_netcdf(provider="nadi", storm_id="2016055S15202")
+        self.assertAlmostEqual(tc_track.data[0].lon.values[0], 203.7, places=1)
+        self.assertAlmostEqual(tc_track.data[0].lon.values[3], -155.7, places=1)
```

Comment:
We check that this storm's IBTrACS data really has this issue before doing the real test below because IBTrACS data might change in the future. It also demonstrates the problem to other people who might look into the tests in the future.

Reply:
It's not a bug. In the IBTrACS specifications, it's not said that the longitude values of the reporting agencies adhere to any specific format. So, it's well possible that this will stay like this forever since it's literally what the agency reported. However, there's also no guarantee that it won't change in the future.

Reply:
With [eaed9cf](https://github.com/CLIMADA-project/climada_python/pull/367/commits/eaed9cf0c6ec965da59958dc8dc340fcead83f7d), the test data is chosen in a more consistent way.

Reply:
looks good